mimic/model/clb_objects.py

```diff
@@ -13,6 +13,8 @@
 
 from six import string_types
 
+from toolz.dicttoolz import dissoc
+
```

Comment:
I see otter uses it, so I'm happy with this. :+1:

Just be careful that we're not in the habit pulling in lots of libraries to do trivial things, as it makes the maintainers have to Google the project and at least read the docs to understand what's going on in your code.